src/Google.Api.Gax/CallTiming.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax
+{
+    public enum CallTimingType
+    {
+        /// <summary>
+        /// Call uses retry, specified with a <see cref="RetrySettings"/>.
+        /// </summary>
+        Retry,
+
+        /// <summary>
+        /// Call does not use retry; call expiration is specified with an <see cref="Google.Api.Gax.Expiration"/> .
+        /// </summary>
+        Expiration,
+    }
+
+    public sealed class CallTiming
```

Comment:
But "the expected timeframe" is part of these settings - so it's not like these settings are *only* used when there's a failure.